theseus/geometry/vector.py

```diff
@@ -204,7 +204,21 @@ def _project_impl(
         return euclidean_grad.clone()
 
     @staticmethod
-    def exp_map(tangent_vector: torch.Tensor) -> "Vector":
+    def exp_map(
+        tangent_vector: torch.Tensor, jacobians: Optional[List[torch.Tensor]] = None
+    ) -> "Vector":
+        if tangent_vector.ndim != 2:
+            raise ValueError("The dimension of tangent vectors should be 2.")
+
+        if jacobians is not None:
```

Comment:
In fact, maybe we can just have `Point2` and `Point3` reuse this method, and maybe just pass a custom error message?

Reply:
Yes, I agree.